ValveResourceFormat/Resource/ResourceTypes/RubikonPhysics/Shapes/Mesh.cs

```diff
@@ -20,12 +20,17 @@ public enum NodeType
         public readonly struct Node
         {
             public readonly Vector3 Min;
+            private readonly uint PackedTypeChildOffset;
 
             /// <summary>
-            /// The 2nd child offset and the node type/split axis.
-            /// Type is stored in the first 2 MSBs.
+            /// The node type/split axis.
             /// </summary>
-            public readonly uint Children;
+            public NodeType Type => (NodeType)(PackedTypeChildOffset >> 30);
+
+            /// <summary>
+            /// The 2nd child offset.
```

Comment:
```suggestion
            /// The 2nd child offset, otherwise when <see cref="Type" /> is <see cref="NodeType.Leaf" />, this is the triangle count.
```